src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -272,35 +273,34 @@ class OPENVINO_GENAI_EXPORTS LLMPipeline {
 OPENVINO_GENAI_EXPORTS std::pair<std::string, Any> streamer(StreamerVariant func);
 OPENVINO_GENAI_EXPORTS std::pair<std::string, Any> generation_config(const GenerationConfig& config);
 
-OPENVINO_GENAI_EXPORTS std::pair<std::string, Any> _draft_model(
+OPENVINO_GENAI_EXPORTS std::pair<std::string, Any> draft_model(
     const std::string& model_path,
-    const std::string& device,
-    const ov::AnyMap& llm_config);
+    const std::string& device = {},
```

Comment:
Why does device have a default value here?

Reply:
in this case device for draft model is taken to be the same as for main model.